runHarness/WorkloadDrivers/AuctionWorkloadDriver.pm

```diff
@@ -1326,7 +1326,7 @@ sub startRun {
 									chop($successStr);
 								}
 								my $metricsStr = ", $successStr, throughput:$tptStr, avgRT:$rtStr";
-								$console_logger->info("   [appInstance: $appInstanceNum] Ended: $nameStr${metricsStr}.");
+								$console_logger->info("   [workload $workloadNum, appInstance: $appInstanceNum] Ended: $nameStr${metricsStr}.");
```

Comment:
Should the workload info only print if num workloads > 1?  As this seems a somewhat unusual case.

Reply:
Yes.  Makes sense.  I will make this change before merging.

Reply:
Actually, I am going merge as-is and open a jira issue for this change.  It is non-trivial and I want to get these fixes in.